internal/provisioner/objects/deployment/deployment.go

```diff
@@ -301,3 +298,32 @@ func contourPodLabels(contour *model.Contour) map[string]string {
 	}
 	return labels
 }
+
+// contourPodAnnotations returns the annotations for contour's pods
+func contourPodAnnotations(contour *model.Contour) map[string]string {
+	annotations := map[string]string{}
+	for k, v := range contour.Spec.ContourPodAnnotations {
+		annotations[k] = v
+	}
+
+	metricsPort := metricsPort
+	if contour.Spec.RuntimeSettings != nil &&
+		contour.Spec.RuntimeSettings.Metrics != nil &&
+		contour.Spec.RuntimeSettings.Metrics.Port > 0 {
+		metricsPort = contour.Spec.RuntimeSettings.Metrics.Port
+	}
+
+	annotations["prometheus.io/scrape"] = "true"
```

Comment:
worth testing+documenting that if the user tries to set these, then we will overwrite them with our values